modules/aws-load-balancer-operator-uninstall.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="aws-load-balancer-operator-uninstall_{context}"]
+= Uninstalling an AWS Load Balancer Operator
+To uninstall an AWS Load Balancer Operator (ALBO) and perform an overall clean up of the associated resources, perform the following procedure.
+
+.Procedure
+. Clean up sample application by deleting the Load Balancers created and managed by the ALBO. For more information, reagrding deleting Load Balancers,  see link:https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-delete.html[Delete an Application Load Balancer].
+. Clean up AWS VPC tags by removing the VPC tags that were added to the subnets for discovering subnets and creating Application Load Balancers (ALBs). For more information, see link:https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-delete.html[Remove tags from resource].
+. Clean up ALBO components by deleting both the ALBO and the Application Load Balancer Controller (ALBC).
+For more information, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/operators/administrator-tasks#olm-deleting-operators-from-a-cluster[Deleting Operators from a cluster].
```

Comment:
See my previous comment about removing xrefs from modules and only linking to other parts of the docs by using xrefs in assemblies.